frontend/src/leader-list.tsx

```diff
@@ -49,16 +53,27 @@ function colorForIndex(index: number): string {
 }
 
 export function getLeaders(pulls: Pull[], extractSigsFromPull: (pull: Pull) => Signature[]) {
-   const signatures: Signature[] = pulls.flatMap(extractSigsFromPull);
-   const users = signatures.map((signature) => signature.data.user);
-   const groupedByUsers = users
-      .reduce((grouped: Map<number, SigCount>, user: SignatureUser) => {
-         const group = grouped.get(user.id) || {user, count:0};
-         group.count++;
-         grouped.set(user.id, group)
-         return grouped;
-      }, new Map())
-      .values();
-   return Array.from(groupedByUsers)
+   const recent = toDateString(new Date(Date.now() - 86400 * 1000 * DAYS_TO_EXAMINE));
+   const recentPulls = pulls.filter((pull) => {
+      return pull.merged_at ? pull.merged_at > recent : true;
```

Comment:
Minor point, but I'm reading this as: Look at all PRs that were merged in the past 14 days, or that do not have a merged_at value. I think this means that the leaderboard will also take into account CR stamps on open PRs that have stuck around longer than 14 days.

Reply:
> . I think this means that the leaderboard will also take into account CR stamps on open PRs that have stuck around longer than 14 days.

You are correct. That was intentional. We mostly want "all recent CRs" regardless of when the pull was opened / merged. I just decided to trim the old merged pulls for performance.